### DISCUSSION TITLE
Is the library still working?

### DISCUSSION THREAD

**Discussion by CD-56** [Lioydiano/Classeviva#19] [Domande e Risposte] [answered]
I tried installing the library but it's not recognized in vs code.

**Comment by FLAK-ZOSO** [comment]
Well, what does pip tell you? Because sometimes it's just VS Code intellisense not recognizing the libraries.

**Reply by CD-56** [reply]
the pip says that requirements are already satisfied but in vs code pylance does not recognize the library and the code does not work

**Reply by CD-56** [reply]
![image](https://github.com/Lioydiano/Classeviva/assets/164915367/100af4b5-651c-49b8-b604-0f3084d4ca4e)

**Reply by FLAK-ZOSO** [reply]
Well, for the intellisense try clicking on the "classeviva" green word.

For the code... as long as it runs I guess it's ok, do you get any traceback?

**Reply by CD-56** [reply]
it does not run:
Traceback (most recent call last):
  File "[directory]", line 1, in <module>
    import classeviva
ModuleNotFoundError: No module named 'classeviva'

**Reply by FLAK-ZOSO** [reply, marked as answer]
Ah, well, that's a problem, but I guess it depends on the fact that you have multiple Python environments on your device.

Try with `python -m pip install classeviva.py` or try reloading the terminal I guess.
